scala-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/scala/ScalaInterpreter.scala

```diff
@@ -48,7 +48,7 @@ class ScalaInterpreter(private val config:Config = ConfigFactory.load) extends I
 
    protected val lastResultOut = new ByteArrayOutputStream()
 
-   protected val multiOutputStream = MultiOutputStream(List(Console.out, lastResultOut))
+   protected val multiOutputStream = MultiOutputStream(List(lastResultOut))
```

Comment:
multiOutputStream is overridden by a test class that requires the list, that was the cause of the compilation failure. Your PR should have the same failure.